lib/internal/util/inspect.js

```diff
@@ -541,8 +541,7 @@ function getConstructorName(obj, ctx, recurseTimes, protoProps) {
     const descriptor = ObjectGetOwnPropertyDescriptor(obj, 'constructor');
     if (descriptor !== undefined &&
         typeof descriptor.value === 'function' &&
-        descriptor.value.name !== '' &&
-        tmp instanceof descriptor.value) {
+        descriptor.value.name !== '') {
```

Comment:
This change broke tests inspecting the prototype object, which is not an instance of its constructor.